### ISSUE TITLE
CMS-CI Network Downloads Very Slow

### ISSUE THREAD

**Issue by ElijahLynn** [department-of-veterans-affairs/va.gov-cms#1044]
Small downloads are taking ~5 seconds. 

![image](https://user-images.githubusercontent.com/1504756/75192112-799e6500-5708-11ea-80aa-9a5fe400e7f3.png)

**Comment by ElijahLynn** [comment]
Specific to this instance it appeared. I restarted instance, same symptom. I created new instance but not in the same subnet/AZ and it was fast. Then while I was working on putting another one in the same subnet/AZ, the problemeatic host started performing well. Closing this for now: `for x in {0..100}; do (time curl https://example.com) &> >(grep real); sleep 1;  done`

![image](https://user-images.githubusercontent.com/1504756/75195012-03046600-570e-11ea-897b-5a6cc5d65e30.png)
